sdk/identity/identity/review/identity.api.md

```diff
@@ -135,7 +135,7 @@ export { GetTokenOptions }
 export class InteractiveBrowserCredential implements TokenCredential {
     constructor(options?: InteractiveBrowserCredentialOptions);
     getToken(scopes: string | string[], options?: GetTokenOptions): Promise<AccessToken | null>;
-}
+    }
```

Comment:
Why did this move?

Reply:
I wonder if it's an api extractor thing

Reply:
Yeah it's an API extractor thing. This wasn't linting nor prettier.